### DISCUSSION TITLE
Do you guys need CI ?

### DISCUSSION THREAD

**Discussion by melezhik** [kamalshkeir/kago#8] [Show and tell]
Hi! I am building a free CI service with many languages support including golang. I've noticed you don't have CI. You can use https://ci.sparrowhub.io for that. An example of golang project build (go async+redis service) - https://ci.sparrowhub.io/report/1453

SparrowCI allows to create integration tests with mysql/psql/etc ... Also (I am not sure if this is what you need, but anyway) it's possible to run CI tests in archlinux/debian/ubuntu/alpine containers 

Let me know if this sounds interesting ...

**Comment by kamalshkeir** [comment]
Hi @melezhik , why not, i will give it a try
